plugin/action/throttle/throttle.go

```diff
@@ -45,7 +68,7 @@ type Config struct {
 	//> The event field which defines the time when event was fired.
 	//> It is used to detect the event throughput in a particular time range.
 	//> If not set, the current time will be taken.
-	TimeField  cfg.FieldSelector `json:"time_field" default:"time"` //*
+	TimeField  cfg.FieldSelector `json:"time_field" default:"" parse:"selector"` //*
```

Comment:
why default value is removed?

Reply:
It wasn't work at all. 
Let's set it this way: `json:"time_field" default:"time" parse:"selector"`